drivers/spi/spi_ll_stm32.c

```diff
@@ -55,6 +55,11 @@ static int spi_stm32_get_err(SPI_TypeDef *spi)
 	if (sr & SPI_STM32_ERR_MSK) {
 		SYS_LOG_ERR("%s: err=%d", __func__,
 			    sr & SPI_STM32_ERR_MSK);
+
+		/* OVR error must be explicitly cleared */
+		if (LL_SPI_IsActiveFlag_OVR(spi))
```

Comment:
you forgot the {}  these are mandatory in Zephyr even for on liner statements.

Reply:
Yes I know. But still I always forget....
:(